src/services/functionAppService.test.ts

```diff
@@ -107,8 +107,8 @@ describe("Function App Service", () => {
 
   it("gets master key", async () => {
```

Comment:
suggestion: rename test description to start with `should/should not`

Reply:
will do